### ISSUE TITLE
Does this package works with `electron-updater` package?

### ISSUE THREAD

**Issue by alisherafat01** [ArekSredzki/electron-release-server#363]
I want to know which package to use: 
```
import { autoUpdater } from 'electron-updater';
or 
import {  autoUpdater } from 'electron';
```

**Comment by lilhuy0405** [comment]
Yes it does you can import it like this `import {autoUpdater} from 'electron';`
And follow this docs: https://www.electronjs.org/docs/latest/tutorial/updates#using-other-update-services
But if you follow this docs there is a note: the platform's name saved in electron-release-server and the `process.platform` is difference.
So this code from docs will build a wrong update url
```
const server = 'https://your-deployment-url.com'
const url = `${server}/update/${process.platform}/${app.getVersion()}`

autoUpdater.setFeedURL({ url })
```
In my case `proccess.platfrom` is win64 but the platform that release server expected is something looks like `windows_64` so we need to format the platform to match the release server
